grid-proxy/pkg/client/grid_client.go

```diff
@@ -5,12 +5,16 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"net/url"
 	"strconv"
 
+	"github.com/gorilla/schema"
 	"github.com/pkg/errors"
 	"github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/types"
 )
 
+var encoder = schema.NewEncoder()
```

Comment:
why it's defined here nit in the function?

Reply:
declared as a global variable. it is being used in the client_test methods